.github/workflows/workflow.yml

```diff
@@ -516,15 +516,15 @@ jobs:
           set_gcloud_config "$GCP_PROJECT" "$GCP_ZONE" "$GKE_CLUSTER"
         )
         . "$dir/path.bash.inc"
-    - name: Helm chart creation
-      if: startsWith(github.ref, 'refs/tags')
-      run: |
-        bin/helm-build package
-    - name: Edge Helm chart upload
+    - name: Edge Helm chart creation and upload
       if: startsWith(github.ref, 'refs/tags/edge')
       run: |
+        gsutil rsync cp gs://helm.linkerd.io/edge/index.yaml target/helm/index-pre.yaml
```

Comment:
what is the `cp` subcommand to `rsync`?  I can't find it on https://cloud.google.com/storage/docs/gsutil/commands/rsync and it does not work for me locally.

Reply:
Good catch. It should be just `gsutil cp`.